app/controllers/concerns/bbb_helper.rb

```diff
@@ -192,7 +192,12 @@ def meeting_start_time
   end
 
   def bigbluebutton_moderator_roles
-    Rails.configuration.bigbluebutton_moderator_roles.split(',')
+    roles_params = bbb_moderator_roles_params(@room.tenant)
+    if roles_params && roles_params.present?
+      roles_params
+    else
+      Rails.configuration.bigbluebutton_moderator_roles.split(',')
+    end
```

Comment:
this block of code is not wrong, but you can write it more simply:
the `.present?` method actually also checks for both nil and emptiness so you can replace line 196 with just `if roles_params.present?`

however, there's also another method in rails called `presence`, which returns the object if it's present. With this method, you can replace the entire block of code with something like this:
`roles_params.presence || Rails.configuration.bigbluebutton_moderator_roles.split(',')`